core/foundation/src/FoundationUtils.cxx

```diff
@@ -88,5 +89,89 @@ std::string MakePathRelative(const std::string &path, const std::string &base, b
    }
    return result;
 }
+
+/// Transforms a file path by replacing its backslashes with slashes.
+void ConvertToUnixPath(std::string& Path) {
+   std::replace(Path.begin(), Path.end(), '\\', '/');
+}
+
+const char* GetFallbackRootSys() {
+#ifdef WIN32
+   static char lpFilename[_MAX_PATH];
+   if (::GetModuleFileNameA(
+          NULL,                   // handle to module to find filename for
+          lpFilename,             // pointer to buffer to receive module path
+          sizeof(lpFilename))) {  // size of buffer, in characters
+      auto parent_path = [](std::string& path) {
+         return path.substr(0, path.find_last_of("/\\"));
+      };
+      return StrDup(parent_path(parent_path(lpFilename)).c_str());
+   }
+   return nullptr;
+#else
+   return "/usr/local/root";
```

Comment:
This is wrong. It should return `${CMAKE_INSTALL_PREFIX}`.

Reply:
This is moving code around. I will fix this in the next commit.